setup-registry/action.yaml

```diff
@@ -26,4 +31,4 @@ runs:
     - name: Run registry
       shell: bash
       run: |
-        PORT=${{inputs.port}} crane registry serve &
+        PORT=${{inputs.port}} crane registry serve --blobs-to-disk=${{inputs.disk}} &
```

Comment:
This will only work with `tip` 😞

Reply:
Documented.

Reply:
My point is that even `--blobs-to-disk=false` will break without `tip`